build/rust/Cargo.lock

```diff
@@ -1715,6 +1801,25 @@ version = "0.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "8b870d8c151b6f2fb93e84a13146138f05d02ed11c7e7c54f8826aaaf7c9f184"
 
+[[package]]
+name = "ppoprf"
+version = "0.0.1"
```

Comment:
This is an invalid version number and is going to cause issues for gnrt https://chromium.googlesource.com/chromium/src/+/HEAD/tools/crates/
You cannot have a minor version of 0 https://semver.org/
also we already have two different version of base64 so can we use 0.13.0 here to match adblock?